style.css

```
@@ -691,7 +706,21 @@ body.dark-mode .icon-sun {
   transition: background-color 140ms ease, color 140ms ease, box-shadow 140ms ease, opacity 140ms ease;
 }
 
-.discovery-marker span {
+.discovery-marker {
+  --milestone-color: var(--discovery);
+  --milestone-soft: var(--discovery-soft);
+  border-radius: 50%;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)